lib/bootstrapper/runtime/Dockerfile

```diff
@@ -1,11 +1,8 @@
 FROM lambci/lambda:build-python3.8
 
-ARG PGSTAC_VERSION
-RUN echo "Using PGSTAC Version ${PGSTAC_VERSION}"
-
 WORKDIR /tmp
 
-RUN pip install httpx psycopg[binary,pool] pypgstac==${PGSTAC_VERSION} -t /asset
+RUN pip install httpx psycopg[binary,pool] pypgstac==0.6.13 -t /asset
```

Comment:
Why is this hardcoded now?

Reply:
oh I see, now the user cannot `just` specify a pgstac version but has to provide a full docker build